front/src/modules/pathfinding/hook/usePathfinding.ts

```diff
@@ -248,20 +256,26 @@ export const usePathfindingV2 = (
                   positionOnPath: pathfindingResult.path_items_positions[i],
                   stopFor,
                   ...(correspondingOp && {
+                    name: correspondingOp.name,
+                    uic: correspondingOp.uic,
+                    ch: correspondingOp.ch,
```

Comment:
I'm not sure why you would want to erase these current op properties by the pathSteps ones ? They should be the same no ? 
We needed that for kp and coordinates because between two pathfindings, the path could sometimes use different tracks.

Reply:
As discussed, we can keep this like this :+1: